codespell_lib/data/dictionary.txt

```diff
@@ -56538,6 +56544,8 @@ thaughts->thoughts
 thawin->thawing, thaw in,
 thay->they, that,
 thck->thick
+thead->thread
+theads->threads
```

Comment:
I think these should stay in `code`, they are commonly used variable names. ahead, bhead, thead, etc.

Reply:
Also it is valid HTML:
https://www.w3schools.com/tags/tag_thead.asp

Reply:
OK for `thead`. But `theads`?

Reply:
I feel like I hit theads as a plural of thead, but it seems it just came in via https://github.com/codespell-project/codespell/commit/defc684225e9341db67d10c681889413effe6287#diff-f6d3c2a2b3be34e25a8bb6430ae4d0eb8ec250cf96a4d0631635e8c30e44ea94 and https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/plain/scripts/spelling.txt

Reply:
I put it back to _code_.